src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -392,6 +409,13 @@ impl Program {
     pub fn get_variable(&self, name: &str) -> Option<&VariableDeclarator> {
         for item in &self.body {
             match item {
+                BodyItem::ImportStatement(_stmt) => {
+                    // TODO: Should we do this?
+                    // if stmt.identifier() == name {
+                    //     return Some(stmt);
+                    // }
```

Comment:
Makes sense to do it to me

Reply:
Done. 8358f64ef7ce4a3160280093974340b11c5c352a